clusterctl/testdata/create-cluster-no-args-invalid-flag.golden

```diff
@@ -3,15 +3,16 @@ Usage:
   clusterctl create cluster [flags]
 
 Flags:
-  -a, --addon-components string      A yaml file containing cluster addons to apply to the internal cluster
-      --cleanup-external-cluster     Whether to cleanup the external cluster after bootstrap (default true)
-  -c, --cluster string               A yaml file containing cluster object definition
-  -h, --help                         help for cluster
-      --kubeconfig-out string        Where to output the kubeconfig for the provisioned cluster (default "kubeconfig")
-  -m, --machines string              A yaml file containing machine object definition(s)
-      --provider string              Which provider deployment logic to use (google/vsphere/azure)
-  -p, --provider-components string   A yaml file containing cluster api provider controllers and supporting objects
-      --vm-driver string             Which vm driver to use for minikube
+  -a, --addon-components string              A yaml file containing cluster addons to apply to the internal cluster
+      --cleanup-external-cluster             Whether to cleanup the external cluster after bootstrap (default true)
+  -c, --cluster string                       A yaml file containing cluster object definition
+  -k, --existing-cluster-kubeconfig string   path to an existing cluster's kubeconfig (intead of using minikube)
```

Comment:
Does this need to use the -k? Since it's not a required flag, it seems like we could force people to use the full "existing-cluster-kubeconfig" string